application/src/main/data/upgrade/3.1.0/schema_update.sql

```diff
@@ -0,0 +1,27 @@
+--
+-- Copyright © 2016-2020 The Thingsboard Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+DROP TABLE IF EXISTS entity_profile;
+
+CREATE TABLE IF NOT EXISTS entity_profile (
+    id              varchar(31) NOT NULL CONSTRAINT entity_profile_pkey PRIMARY KEY,
+    name            varchar(255),
+    tenant_id       varchar(31),
+    entity_type     varchar(255),
+    profile         varchar,
+    additional_info varchar,
+    CONSTRAINT name_tenant_id_type_unq_key UNIQUE (name, tenant_id, entity_type)
```

Comment:
I think that constraint should be (tenant_id, entity_type, name). In such case the automatic index is more usable. We always know tenant_id and entity_type when executing find queries. But Name may have different conditions (IN, LIKE, etc). This needs to be changed in the main SQL file as well.

Reply:
agree, fixed